codegen/src/api/mod.rs

```diff
@@ -82,6 +82,7 @@ where
         derives,
         type_substitutes,
         crate_path,
+        true,
```

Comment:
I reckon we should make sure that the macro doesn't generate any docs, in this case (since they are sortof unnecessary there by default). We can always expose a flag in the macro args then to add docs if IDE's can ever learn to ebnefit from it :)

Reply:
That makes total sense! I've disabled by default the documentation generation for the `subxt` macro and added a flag to its attributes: `expose_documentation`, that when present the generated code will expose the metadata documentation